packages/travix-ui-kit/components/calendar/calendar.js

```diff
@@ -289,12 +289,15 @@ Calendar.propTypes = {
 
   /**
    * Locale definitions, with the calendar's months and weekdays in the right language.
-   * Also contains the startWeekDay which defines in which week day starts the week.
+   * It also contains the startWeekDay which defines in which week day starts the week,
+   * dateOrder which defines date order and specific year symbol which used in date for some Asian markets.
    */
   locale: PropTypes.shape({
+    dateOrder: PropTypes.object,
```

Comment:
Shouldn't it be `string`?

Reply:
Yep. It should be a string. Thanks for catching it.